04/app.js

```diff
@@ -2,8 +2,20 @@ const user = {
     firstName: 'Adam',
     lastName: 'Nowak',
     born: {
-        day: '14',
-        month: '04',
-        year: '1985'
+        day: '09',
+        month: '05',
+        year: '2024'
     }
+}
+
+const date = new Date();
+const day = date.getDate();
+const month = date.getMonth() +1;
+ 
+const userBirthday = new Date(user.born.year, user.born.month, user.born.day);
+
+if(month === userBirthday.getMonth() & day === userBirthday.getDate()) {
```

Comment:
Tutaj musimy zrobić 2x `&` inaczej jest to operatora bitowy, a nie logiczne i.
Tutaj więcej informacji: https://stackoverflow.com/questions/7310109/whats-the-difference-between-and-in-javascript